src/Fixer/Operator/NotOperatorWithSuccessorSpaceFixer.php

```diff
@@ -50,6 +44,26 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     /**
      * {@inheritdoc}
      */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Logical NOT operators (!) should have one trailing whitespace.',
```

Comment:
Seems like a merging `NotOperatorWithSuccessorSpaceFixer` and `NotOperatorWithSpaceFixer` into a single configurable fixer would make sense.